Sources/ACKLocalizationCore/ACKLocalization.swift

```diff
@@ -140,18 +140,66 @@ public final class ACKLocalization {
         transformValuesPublisher(valueRange, with: config.languageMapping, keyColumnName: config.keyColumnName)
     }
     
+    /// Builds plurals from `rows` of each language
+    ///
+    /// - Parameter `rows`: All translations of the selected language
+    /// - Returns: Plural keys that are specified in `rows`
+    func buildPlurals(from rows: [LocRow]) -> [String: PluralRuleWrapper] {
+        var plurals: [String: PluralRuleWrapper] = [:]
+        rows.forEach {
+            guard
+                // Matches the translation key that contains the predefined plural pattern
+                let patternRange = $0.key.range(of: Constants.pluralPattern, options: .regularExpression),
+                // Loads the base of that translation key
+                let key = $0.key.components(separatedBy: "##").first
+            else { return }
+            
+            let pattern = $0.key[patternRange]
+            
+            // Loads the current plural rule
+            guard let pluralKeyRange = pattern.range(of: Constants.pluralKeyPattern, options: .regularExpression) else { assertionFailure(); return }
```

Comment:
I think you can use IMHO nicer syntax `return assertionFailure`
also should not this throw an errror ? 🤔